docs/development/addons.md

```diff
@@ -112,6 +112,58 @@ class ExtraView(RalphDetailView):
     label = 'Extra Detail View'
 ```
 
+## Override methods (hooks)
+You can change behaviour some parts of Ralph by overriding exposed methods. It all comes down to add a new entry point in your package (entry in ``setup.py``) and configure by changing settings (by exporting a environment variable) for appropriate entry. For your convience, we create management command for checking current configuration for hooks. You can run it by following command:
+
+```bash
+$ dev_ralph show_hooks_configuration
+
+Hooks:
+    back_office.transition_action.email_context:
+            default (active)
+            foo_method
+            bar_method
+```
+
+Where `foo_method` and `bar_method` are defined in `setup.py` in section `entry_points`:
+
+```python3
+from setuptools import setup, find_packages
+
+setup(
+    name='my_package',
+    ...
+
+    entry_points={
+        'back_office.transition_action.email_context': [
+            'foo_method = my_package.helpers:get_foo_context',
+            'bar_method = my_package.helpers:get_bar_context',
+        ]
+    },
+    ...
+)
+
+```
+
+
+### Available hooks
+| Name                                          | Description                                         |
+| --------------------------------------------- | --------------------------------------------------- |
+| `back_office.transition_action.email_context` | returns subject and body based on transition's name |
```

Comment:
Maybe could we add new column with arguments that are passed to given context and expected return type/format? So a person who wants override some method is not forced to dive into source code?